ghost/core/test/unit/server/services/lib/magic-link/index.test.js

```
@@ -69,21 +69,22 @@ describe('MagicLink', function () {
                     id: '420'
                 },
                 type: 'blazeit',
-                referrer: 'https://whatever.com'
+                referrer: 'https://whatever.com',
```

ESLint

[error] Unexpected trailing comma.

(comma-dangle)